Source/UINavigation/Private/UINavWidget.cpp

```diff
@@ -594,6 +591,8 @@ void UUINavWidget::UINavSetup()
 	{
 		OnPromptDecided(PromptWidgetClass, PromptSelectedIndex);
 	}
+
+	bCompletedSetup = true;
```

Comment:
This should be true before OnSetupCompleted is called

Reply:
ok